server/routes/checkInformationRoutes.ts

```diff
@@ -75,22 +70,11 @@ export default class CheckInformationRoutes {
   }
 
   public submitCheckInformation: RequestHandler = async (req, res): Promise<void> => {
-    const { username, caseloads, token } = res.locals.user
+    const { username, token } = res.locals.user
     const { nomsId } = req.params
 
-    const prisonerDetail = await this.prisonerService.getPrisonerDetail(username, nomsId, caseloads, token)
-    const sentencesAndOffences = await this.prisonerService.getActiveSentencesAndOffences(
-      username,
-      prisonerDetail.bookingId,
-      token
-    )
     const userInputs = this.userInputService.getCalculationUserInputForPrisoner(req, nomsId)
-    const errors = await this.calculateReleaseDatesService.validateBackend(
-      nomsId,
-      userInputs,
-      sentencesAndOffences,
```

Comment:
sentencesAndOffences was unused in the called method therefore removed it (ergo the call to get prisonerDetail  became unnecessary)